examples/webcomponents/index.html

```diff
@@ -47,6 +51,14 @@
             ReactDOM.render(<a href={url}>{name}</a>, mountPoint);
           }
         }
+        ,detachedCallback: {
+          value: function() {
+          }
+        }
+        ,attributeChangedCallback: {
```

Comment:
You are correct, they are not required for the purposes of the demonstration, though it is quite nice to have all four callbacks available should someone want to know which ones exist or want to expand upon the example. Also, I didn't see it as complicating the example since they are empty functions but to your point, less code to read is less. I will remove them.